skins/standard/global.css

```diff
@@ -304,7 +306,6 @@ div#docslinks {
 
 /* tbody.file pre is for the Diff view of attachments. */
 .bz_comment_text, .uneditable_textarea, tbody.file pre {
-     font-family: monospace;
```

Comment:
Is this change to bz_comment_text necessary to bring Fira Sans to the User Story field?